### PR TITLE
Fix doc example for fut::ActorFuture

### PR DESCRIPTION
**Pull request by Jonathas-Conceicao**
The example for the use of `ResponseActFuture` on `ActorFuture` doc was outdated for actix v0.9, I've swapped the `ignore` for a `norun` on the doc to make the error actually visible by the build system and updated the example so it can work with the most recent version of Actix.

### CODE REVIEW

File: src/fut/mod.rs [hunk 1]
```diff
@@ -48,7 +48,9 @@ use std::task;
 /// Here is an example of a handler on a single actor, deferring work to another actor, and
 /// then updating the initiating actor's state:
 ///
-/// ```rust,ignore
+/// ```rust,norun
```
Comment (JohnTitor) [inline src/fut/mod.rs]:
```suggestion
/// ```rust,no_run
```

**Comment by JohnTitor** [comment]
Overall looks good except for one nit.

File: src/fut/mod.rs (hunk 1 above)
Comment (Jonathas-Conceicao) [reply src/fut/mod.rs]: Fixed

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**